luma/core/emulator.py

```diff
@@ -198,8 +198,11 @@ class transformer(object):
     """
     def __init__(self, pygame, width, height, scale):
         self._pygame = pygame
+        self._input_size = (width, height)
         self._output_size = (width * scale, height * scale)
-        self.scale = scale
+        self._scale = scale
+        self._led_on = self._pygame.image.load(os.path.join(os.path.dirname(__file__), "images", "led_on.png"))
```

Comment:
I know we don't have, or need to, introduce a fixed width for a line, e.g. 80, but these statements, _led_on and _led_off, are more readable when kept under 80 imo